flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/table/config/BigQueryConnectorOptions.java

```diff
@@ -110,7 +110,7 @@ private BigQueryConnectorOptions() {}
      * Default: 0 - BigQuery decides the optimal amount.
      */
     public static final ConfigOption<Integer> MAX_STREAM_COUNT =
-            ConfigOptions.key("read.streams.maxcount")
+            ConfigOptions.key("read.streams.max-count")
```

Comment:
These changes will break existing users. Please mention this change in release notes